source/agora/common/BlockStorage.d

```diff
@@ -149,19 +149,24 @@ public class BlockStorage
 
     /***************************************************************************
 
-        Close memory mapped file.
+        Release memory mapped file.
 
     ***************************************************************************/
 
     public void release () @trusted
     {
-        if (this.file is null)
-            return;
         import core.memory : GC;
 
-        destroy(this.file);
-        GC.free(&this.file);
-        this.file = null;
+        try
```

Comment:
Did you see `release` throw ? Because it should never happen.

Reply:
fixed